section3/exercises/loops-arrays.js

```diff
@@ -18,32 +18,47 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
-
-
+for (var i =0; i < animals.length; i++) {
```

Comment:
Pay close attention to the whitespace (spaces, tabs) that you use in your code. See your `for` loops in this file for examples of how you're doing it differently in a lot of places.

Some of this is preference, but your instructors will either define the style you should use and evaluate based on that, or if it's not explicitly defined, will expect you to follow the same styles throughout a project.

Consistent code style makes it easier for other developers to read our code, so it's extremely important.